src/main/java/nextstep/subway/fare/application/FareService.java

```diff
@@ -0,0 +1,26 @@
+package nextstep.subway.fare.application;
+
+import nextstep.subway.fare.domain.discount.DiscountPolicy;
+import nextstep.subway.fare.domain.fare.OverFarePolicy;
+import nextstep.subway.member.domain.AgeGrade;
+import nextstep.subway.path.domain.ShortestPath;
+import org.springframework.stereotype.Service;
+
+@Service
+public class FareService {
+    private static final long DEFAULT_FARE = 1250;
```

Comment:
정적 필드와 인스턴스 필드 사이에 들여쓰기가 있으면 좋을 것 같아요 :)
```suggestion
    private static final long DEFAULT_FARE = 1250;
    
```